### PR TITLE
Diwali webpage

### PR DESCRIPTION
**Pull request by Ashok-mangali**
let see have my code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,16 @@ please
 
 
 
+
```
Comment (kishorekumar646) [inline README.md]: Please remove spaces

File: Tourism_Places/tourism.html [hunk 2]
```diff
@@ -1,10 +0,0 @@
-<!DOCTYPE html>
```
Comment (kishorekumar646) [inline Tourism_Places/tourism.html:1]: Why this code was deleted??

File: Tourism_Places/tourism_places.html [hunk 3]
```diff
@@ -1,9 +0,0 @@
-<!DOCTYPE html>
-<html>
```
Comment (kishorekumar646) [inline Tourism_Places/tourism_places.html:2]: Same above PR note

File: diwali_web page/diwali_web.html [hunk 4]
```diff
@@ -2,8 +2,8 @@
 <html>
     <head>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
-    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
-    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBn+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
```
Comment (kishorekumar646) [inline diwali_web page/diwali_web.html]: Why scripts are updated??

File: diwali_web page/diwali_web.html [hunk 5]
```diff
@@ -2,8 +2,8 @@
...
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
```
Comment (kishorekumar646) [inline diwali_web page/diwali_web.html]: Please test this code was running or not??

**Review by kishorekumar646 — CHANGES_REQUESTED** [review]
Please solve the review comments

**Review by kishorekumar646 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Ashok-mangali) [reply README.md]: done

**Review by kishorekumar646 — APPROVED** [review]

**Comment by kishorekumar646** [comment]
It's npt working

**Pull request closed without merging**